install_dependencies.sh

```diff
@@ -68,27 +71,39 @@ mkdir -p $dependencies_dir && cd $dependencies_dir
 
 echo "installing libodb"
 wget http://codesynthesis.com/download/odb/2.4/libodb-2.4.0.tar.bz2 -O /tmp/libodb.tar.bz2
-tar jxvf /tmp/libodb.tar.bz2
-cd libodb-2.4.0 && ./configure && make && cd ..
+tar jxf /tmp/libodb.tar.bz2
+cd libodb-2.4.0 && ./configure && make
+RESULT=$?
+INSTALL_FAILED=$(( $INSTALL_FAILED + $RESULT ))
```

Comment:
Will a bitwise-or be better here? I am guessing bash won't return negative exit codes, so we are covered here, right?

Reply:
please suggest a syntax because I can't make the "or" work.

Reply:
actually, I realised there's a `set -e` at the top that will make the script exit on the first failure, so I will leave only the "everything ok" at the end, as the "install failed" message will never show